### DISCUSSION TITLE
Is there a function for continuous network usage monitoring?

### DISCUSSION THREAD

**Discussion by bempfoobq052021** [obsproject/obs-studio#4733] [Development Questions] [answered]
Hi, I want to know if there's a function that allows for continuous network usage monitoring?
I want to draw it to a line-chart for monitoring OBS's network usage.
Or is there a plugin that can do that already? Because I can't find it..

**Comment by WizardCM** [comment]
For the most part, OBS offloads network management to the operating system. It doesn't really do any tracking or logging outside of dropped frames.

**Comment by Fenrirthviti** [comment, marked as answer]
Network monitoring is something well outside scope of what OBS is providing. There are several third party tools that will monitor network usage on your system.
